modules/apps/27-interchain-accounts/controller/keeper/migrations_test.go

```diff
@@ -16,15 +20,23 @@ func (suite *KeeperTestSuite) TestAssertChannelCapabilityMigrations() {
 			func() {},
 			true,
 		},
+		{
+			"channel with different port is filtered out",
+			func() {
+				portIdWithOutPrefix := ibctesting.MockPort
+				suite.chainA.GetSimApp().IBCKeeper.ChannelKeeper.SetChannel(suite.chainA.GetContext(), portIdWithOutPrefix, ibctesting.FirstChannelID, channeltypes.Channel{
+					ConnectionHops: []string{ibctesting.FirstConnectionID},
+				})
```

Comment:
We normally favour using `ID` over `Id` in naming (except proto generated code).

```suggestion
				portIDWithOutPrefix := ibctesting.MockPort
				suite.chainA.GetSimApp().IBCKeeper.ChannelKeeper.SetChannel(suite.chainA.GetContext(), portIDWithOutPrefix, ibctesting.FirstChannelID, channeltypes.Channel{
					ConnectionHops: []string{ibctesting.FirstConnectionID},
				})
```

Reply:
We only favor "Id" in proto naming because using "ID" messes with the grpc gateway generation (I'm not sure why nor if this is still an issue, but it was when we initially switched to proto)